### ISSUE TITLE
Tokovoip can't talk after radio talk

### ISSUE THREAD

**Issue by Sammy232** [Itokoyamato/TokoVOIP_TS3#288]
When I spark I can't talk anymore and I have the latest version of tokovoip

**Comment by blaqqendwhyt** [comment]
yes, this guy do the dead person, it's been a while since we know this error, and the way to solve it, as it's requested on the pulls, but nothing change...
